### DISCUSSION TITLE
User Icon request

### DISCUSSION THREAD

**Discussion by pguptaDS** [twilio-labs/paste#870] [Request: Icon] [answered]
May I please request a gender-neutral user icon for RBAC designs? Something like this:

(This comes up when you search for "Profile" on Streamline)
![image](https://user-images.githubusercontent.com/64559618/96546771-b7379b80-125f-11eb-99ae-1365105bde6a.png)

We will be using it in a table like this:
![image](https://user-images.githubusercontent.com/64559618/96546861-ea7a2a80-125f-11eb-998f-5609735f1901.png)

**Comment by serifluous** [comment]
Added to our backlog 👍  Will follow up here when the icon's added.

**Reply by pguptaDS** [reply]
Awesome, thank you!

**Comment by johannaweintraub** [comment]
@pguptaDS Can we also use this icon for User settings page?

**Reply by pguptaDS** [reply]
@johannaweintraub Sure! Where specifically did you want to use this for the user settings page? As the profile pic that we show if you haven't uploaded yours?

**Comment by johannaweintraub** [comment]
It would appear in the console.zen collapsed nav for user settings. See design here: https://share.goabstract.com/64623969-c13a-4ce9-904d-34719d8c4b33?mode=design&sha=d7754fe973cf4b69bc25890e409fa8644004940b

**Comment by pguptaDS** [comment]
Got it, makes sense.

**Comment by johannaweintraub** [comment]
@serifluous Do you know what the timeline is for this? We're building this nav now, but could use a placeholder icon for now if necessary.

**Reply by serifluous** [reply]
I'll send you the SVG, and an engineer on your team can add it to Paste (https://paste.twilio.design/icons/how-to-add-an-icon#self-service-instructions) to speed up the timeline.

**Comment by serifluous** [comment, marked as answer]
Hey @pguptaDS and @johannaweintraub ! UserIcon is available in Paste now: https://paste.twilio.design/icons/

The icon isn't available yet in Sketch, though. We're going to merge it in during our next release (Nov 10).

**Comment by johannaweintraub** [comment]
Thank you!
